src/main/java/net/dries007/tfc/api/calendar/CalendarTFC.java

```diff
@@ -0,0 +1,347 @@
+/*
+ * Work under Copyright. Licensed under the EUPL.
+ * See the project README.md and LICENSE.txt for more information.
+ */
+
+package net.dries007.tfc.api.calendar;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.network.PacketBuffer;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.world.GameRules;
+import net.minecraft.world.dimension.DimensionType;
+import net.minecraft.world.server.ServerWorld;
+import net.minecraftforge.common.util.INBTSerializable;
+import net.minecraftforge.fml.event.server.FMLServerStartingEvent;
+import net.minecraftforge.fml.network.PacketDistributor;
+
+import net.dries007.tfc.network.CalendarUpdatePacket;
+import net.dries007.tfc.network.PacketHandler;
+
+public final class CalendarTFC implements INBTSerializable<CompoundNBT>
```

Comment:
```suggestion
public final class Calendar implements INBTSerializable<CompoundNBT>
```
For naming consistency - if the prefix is required to avoid import collisions, `TFCCalendar` should be used.